includes/Modules/Reader_Revenue_Manager.php

```diff
@@ -418,17 +418,6 @@ protected function setup_assets() {
 			),
 		);
 
-		if ( Feature_Flags::enabled( 'rrmModuleV2' ) ) {
-			$assets[] = new Script(
-				'googlesitekit-reader-revenue-manager-block-editor',
-				array(
-					'src'           => $base_url . 'js/googlesitekit-reader-revenue-manager-block-editor.js',
-					'dependencies'  => array(),
-					'load_contexts' => array( Asset::CONTEXT_ADMIN_POST_EDITOR ),
-				)
-			);
-		}
-
```

Comment:
@tofumatt The suggested approach for registering and enqueuing blocks in this PR is great!

However, besides blocks, this asset in RRM is also designed to add a plugin sidebar panel to the block editor. In this case, would you suggest we also do that in the `blocks` folder, or can we continue using the approach being removed here?

CC: @eugene-manuilov @zutigrm @techanvil

Reply:
Thanks @nfmohit, good spot here.

@tofumatt, it would be preferable not to entirely remove the RRM block editor entry point in this PR, and rather either keep it as-is, or move it to a new location in keeping with the changes introduced in this PR. We'll be building on the RRM entry point in the issue Nahid linked above, #9962.

Reply:
Noted, in that case I'll keep the entry point around, though if it's a Gutenberg block/block editor code I'd suggest moving it to its own `blocks/reader-revenue-manager` location in that PR. But I won't change anything else here for now to keep the PR more tightly scoped.